### PR TITLE
[Frontend] Adds a categories section to the home page.

### PR DESCRIPTION
**Pull request by CamronLDNF**
## Description
Adds an Explore Categories section to the home page.

## PivotalTracker
[Create Explore Categories section on Home page](https://www.pivotaltracker.com/story/show/163727009)

## Tasks
* Create ExploreCategories component
* Write acceptance and component tests

## Screenshots 
screenshot 2019-02-06 at 13 15 42

### CODE REVIEW

File: client/we_meet_client/src/components/ui-components/ExploreCategories.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    let categories = this.state.categories;
+    let categoriesList = categories.map(category => {
+      return (
+        <Box inlineBlock>
+          <Card
+            className="card category-card"
+            key={category.id}
+            border
+            shadow
+            maxW="sm"
```
Comment (Jon1969Edwards) [inline client/we_meet_client/src/components/ui-components/ExploreCategories.js:34]: Should some of this styling be stored in another file?  (tailwind.css)

**Review by shahin1290 — APPROVED** [review]
LGTM

File: client/we_meet_client/src/components/ui-components/ExploreCategories.js (hunk 1 above)
Comment (CamronLDNF) [reply client/we_meet_client/src/components/ui-components/ExploreCategories.js:34]: These are component properties that come with the imported component and have to reside here

Comment (Jon1969Edwards) [reply client/we_meet_client/src/components/ui-components/ExploreCategories.js:34]: Cool.  I figured but thought it was worth mentioning.

**Review by Jon1969Edwards — APPROVED** [review]
I approved this but then realized semaphore isn't passing.  I assume this will be fixed.

**Pull request merged**